### ISSUE TITLE
Sixel raster attributes not recognized.

### ISSUE THREAD

**Issue by skipper1947** [mkrueger/icy_term#20]
If I change the "pixels per sixel" attribute in a sixel file from 1 to 2 (doubling the displayed size), it does seem to be recognized in Icy Term. FYI this is the relevant excerpt from the CTerm document (note the " character precedes the attribute fields):
		" p1 ; p2 [ ; p3 [ ; p4 ] ]
			Raster Attributes
			p1 indicates the vertical size in pixels of each
			sixel.  p2 indicates the horizontal size in pixels.
			p3 and p4 define the height and width (in sixels)
			respectively of a block to fill with the background
			colour.  This block may not extend past the current
			bottom of the screen.  If any pixel data characters
			proceed this command, it is ignored.

**Comment by mkrueger** [comment]
Should work now. I've refactored the sixel backend a lot - was more or less just a test implementation. Didn't think anyone really used that for something.

I wrote some unit tests, but still: expect some bugs. I'll upload 5.3 if the build bot runs through.

side note: I've fixed the xterm extended color palette. No idea why the old palette was from but your BBS showed that the palette I used was wrong :). TY

**Comment by skipper1947** [comment]
Thanks for all your work! I can't guarantee that anybody but me is using the pixels per sixel property. :)  I keep strutting out new stuff I discover in SyncTERM, hoping to encourage anybody else by showing what can be done (not that I am any sort of expert, I am just retired, so have lots of time to monkey with it)
So far I have had one guy who actually asked how I did something (implementing custom graphics via downloadable fonts). He actually did it and added some stuff to his site, that was gratifying.
Skip
    On Saturday, August 12, 2023 at 12:55:44 PM PDT, Mike Krüger ***@***.***> wrote:  
 
 


Should work now. I've refactored the sixel backend a lot - was more or less just a test implementation. Didn't think anyone really used that for something.

I wrote some unit tests, but still: expect some bugs. I'll upload 5.3 if the build bot runs through.

side note: I've fixed the xterm extended color palette. No idea why the old palette was from but your BBS showed that the palette I used was wrong :). TY
